src/content/ui/adaptive-responsive/large-screens.md

```diff
@@ -0,0 +1,237 @@
+to lock screen orientation,
+but some developers lock screen orientation anyway.
+Be aware that this can cause problems when running your
+app on a foldable device.
+
+When running on a foldable, the app might look ok
+when the device is folded. But when unfolding,
+you might find the app letterboxed.
+
+As described in the [SafeArea & MediaQuery][] page,
+letterboxing means that the app's window is locked to
+the center of the screen while the window is
+surrounded with black.
+
+Why can this happen?
+
+This can happen when using `MediaQuery` to figure out
+the window size for yur app. When the device is folded,
+orientation is restricted to portrait mode.
+Under the hood, `setPreferredOrientations` causes
+Android to use a portrait compatibility mode and the app
+is displayed in a letterboxed state.
+In the letterboxed state, `MediaQuery` never receives
+the larger window size that allows the UI to expand.
+
+You can solve this in one of two ways:
+
+* Support all orientations.
+* Use the dimensions of the _physical display_.
+  In fact, this is one of the _few_ situations where
+  you would use the physical display dimensions and
+  _not_ the window dimensions.
+
+How to obtain the physical screen dimensions?
+
+You can use the [`Display`][] API, introduced in
+Flutter 3.22/Dart 3.4, which contains the size,
+pixel ratio, and the refresh rate of the physical device. 
+
+<b>PENDING: Your notes said Flutter 3.13 introduced Display, but isn't it actually Flutter 3.22?</b>
+
+[`Display`]: https://main-api.flutter.dev/flutter/dart-ui/Display-class.html
+
+Here is some sample code for getting a `Display` object:
+
+```dart
+/// AppState object.
+ui.FlutterView? _view;
+
+@override
+void didChangeDependencies() {
+  super.didChangeDependencies();
+  _view = View.maybeOf(context);
+}
+
+void didChangeMetrics() {
+  final ui.Display? display = _view?.display;
+}
+```
+
+The important thing is to find the display of the
+view that you care about. This is a forward-looking
+API that should handle current _and_ future multi-display
+and multi-view devices.
+
+<b>PENDING: What is tricky about selecting the "right" view? It feels like this explanation is incomplete</b>
+
+<b>PENDING: OK, the next thing covered in the slides (starting with slide 66) is an actual example that switches between Dialog implementations. This would be hard to show without providing a specific example. I think it would be VERY useful to include this in the docs. Ditto for the navigation UI (starting with slide 81) and custom layout (slide 96). Thoughts?</b>
+
+[SafeArea & MediaQuery]: /ui/adaptive-responsive/safearea-mediaquery
+
+## Adaptive input
+
+Adding support for more screens, also means
+expanding input controls.
+
+Android guidelines describe three tiers of large format device support.
+
+![3 tiers of large format device support](/assets/images/docs/ui/adaptive-responsive/large-screen-guidelines.png){:width="90%"}
+
+Tier 3, the lowest level of support,
```

Comment:
Apples iPad guidelines also call out handling inputs as a best practice https://developer.apple.com/design/human-interface-guidelines/designing-for-ipados#Best-practices

Reply:
Thanks for providing links to the Apple versions of these rules!